recipe/meta.yaml

```diff
@@ -17,7 +17,7 @@ requirements:
     - cuda-cupti-dev 12.0.90
     - cuda-gdb 12.0.90
     - cuda-nvdisasm 12.0.76
-    - cuda-nvprof 12.0.90
+    - cuda-nvprof 12.0.90         # [not aarch64]
```

Comment:
Skip `cuda-nvprof` on `linux_aarch64` as it is [not built for that OS/architecture]( https://github.com/conda-forge/cuda-nvprof-feedstock/blob/97c9e111ddf4b81f5afd20ff737fd83ba57e13cf/recipe/meta.yaml#L25 ).